fe/fe-core/src/test/java/com/starrocks/analysis/MvChangesAsTableChangesTest.java

```diff
@@ -57,8 +62,18 @@ public void testAsyncIfTableTruncate() throws SQLException {
         TaskManager taskManager = GlobalStateMgr.getCurrentState().getTaskManager();
         List<TaskRunStatus> taskRuns = taskManager.showTaskRunStatus(null);
 
+        int retryCount = 0;
```

Comment:
Maybe awaitility is a better choice. WDYT? We can see https://github.com/awaitility/awaitility.

Reply:
You are right, this library of yours can make the code more elegant, but for now let's put this in because it may block other tests. Later we will change it to a more elegant way of writing.